src/app/models/form.server.model.ts

```diff
@@ -300,6 +301,33 @@ const MultirespondentFormSchema = new Schema<IMultirespondentFormSchema>({
     type: String,
     required: true,
   },
+  workflow: [
+    {
+      workflow_type: {
+        type: String,
+        enum: Object.values(WorkflowType),
+        default: WorkflowType.Static,
+        required: true,
+      },
+      emails: {
+        type: [
+          {
+            type: String,
+            trim: true,
+          },
+        ],
+        set: transformEmails,
+        validate: {
+          validator: (v: string[]) => {
+            if (!Array.isArray(v)) return false
+            return v.every((email) => validator.isEmail(email))
+          },
+          message: 'Please provide valid email addresses',
+        },
+      },
+      //TODO: add form fields
```

Comment:
What are these form fields to be added?

Reply:
These form fields refer to the next stage of static routing, where we specify which fields the respondent should fill in. I'll remove it haha ([053fd8d](https://github.com/opengovsg/FormSG/pull/6968/commits/053fd8d6cb7d0e4a8bda7203032f00041716d711)), it might be confusing. And whoever works on it next should be able to figure it out anyway!